easydata.net/src/EasyData.Core/Services/EntityAttributeMetaBuilder.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+
+namespace EasyData.Services
+{
+    public class EntityAttributeMetaBuilder
+    {
+        /// <summary>
+        /// Define wether the attributed is enabled.
```

Comment:
I think you should use word "property" instead of "attribute" in code and comments.